src/get-started/quickstart-java.md

```diff
@@ -35,7 +35,7 @@ $ mvn archetype:generate \
       -DarchetypeVersion=LATEST \
       -DgroupId=org.example \
       -DartifactId=word-count-beam \
-      -Dversion="0.1" \
+      -Dversion="0.6.0" \
```

Comment:
This seems incorrect -- it is specifying the version of generated project, not the version of the artifact (or version of Beam).

Can you double check your experience here?